ui/components/multichain/account-details/account-details-authenticate.js

```diff
@@ -13,8 +13,9 @@ import {
   ButtonSecondary,
   FormTextField,
   Box,
+  Text,
 } from '../../component-library';
-import { Text } from '../../component-library/text/deprecated';
+
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import { exportAccount, hideWarning } from '../../../store/actions';
 
```

Comment:
Updating to use BannerAlert `description` prop. This will keep the font size at 14px on popup view but 16px on expanded 

### Before
https://github.com/MetaMask/metamask-extension/assets/8112138/27f80329-68e2-4835-9799-b1d835850d4f

### After
https://github.com/MetaMask/metamask-extension/assets/8112138/b71049f9-da50-4da9-a6eb-2fb6be03b95b